core/src/main/scala/filodb.core/downsample/DownsampledTimeSeriesShard.scala

```diff
@@ -204,12 +209,17 @@ class DownsampledTimeSeriesShard(rawDatasetRef: DatasetRef,
         stats.indexEntriesRefreshed.increment(count)
         logger.info(s"Refreshed downsample index with new records numRecords=$count " +
           s"dataset=$rawDatasetRef shard=$shardNum fromHour=$fromHour toHour=$toHour")
+        partKeyIndex.commit()
+        val hourMillis = toHour  * 1000 * 60 * 60
```

Comment:
It would be better to get the max time we pulled from the underlying store to avoid missed data.